sgx-show/src/sink/mod.rs

```diff
@@ -0,0 +1,15 @@
+pub mod bit;
+pub mod debug;
+pub mod mask;
+
+pub trait Sink {
+    type Type;
+
+    fn info(&self, _value: &Self::Type) -> Option<String> {
+        None
```

Comment:
It seems like this will *always* return `None`; is there something I'm missing about how this is used?

Reply:
This is a default function which will be overridden by specific implementations of `Sink`. The default outputs nothing.

Reply:
Ahh, I see. Ex. here https://github.com/enarx/enarx/pull/97/files#diff-28457ac1732bda7b910ee565ab93f102R12

Reply:
Exactly.